src/packages/ThemeGenerator/__tests__/ThemeGenerator.spec.js

```diff
@@ -11,7 +11,7 @@ describe('ThemeGenerator', () => {
                 '--color-brand-30': '#303030',
             });
             const themeGenerator = new ThemeGenerator(OneUITheme);
-            expect(themeGenerator.generate().replace(/\s+/g, '')).toEqual(
+            expect(themeGenerator.generateThemeCss().replace(/\s+/g, '')).toEqual(
```

Comment:
The [test case name](https://github.com/textkernel/oneui/pull/752/files?diff=unified&w=1#diff-bedae70b94b80d663f3399eab9bc1a01d703bbef3fa49b01c361e820a4965d8dR4) should be adjusted 

`describe('#generate()' -> describe('#generateThemeCss()'`